lib/file_validator.rb

```diff
@@ -85,8 +79,8 @@ def expected_files(scope)
       expected_files += tree_tarballs
     else
       expected_files = files.reject do |file|
-        changes_proc.call(file).include?("deleted") || (file["type"] && file["type"] != "file")
-      end.map { |file| file["name"] }
+          file["changes"].include?("deleted") || (file["type"] && file["type"] != "file")
+        end.map { |file| file["name"] }
```

Comment:
`end` at 83, 8 is not aligned with `files.reject do |file|` at 81, 23 or `expected_files = files.reject do |file|` at 81, 6.<br>Avoid multi-line chains of blocks.